### ISSUE TITLE
RFE: Support alias names for checks instead of only numeric ids

### ISSUE THREAD

**Issue by tyll** [koalaman/shellcheck#1948]
#### For new checks and feature suggestions
- [N/A] shellcheck.net (i.e. the latest commit) currently gives no useful warnings about this
- [x] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh

# shellcheck disable=single-quote-expansion
xx='$i_dont_want_to_expand_the_var_here'
```

#### Here's what shellcheck currently says:

It only supports codes like SC2016

#### Here's what I wanted or expected to see:

Being able to specify names instead of codes to make it easier to understand which code is ignored.
